nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml

```diff
@@ -0,0 +1 @@
+nelmio_api_doc: ~
```

Comment:
And by the way, maybe add some default values or comments about values that may be change by the end-developer is a good idea. Having an almost-empty file is not really a good practice